pkg/phlaredb/query/iters_test.go

```diff
@@ -394,20 +409,35 @@ func TestBinaryJoinIterator(t *testing.T) {
 			name:                "first two time stamps each",
 			expectedResultCount: 2 * 8, // expect two profiles for each series
 			timePredicate:       NewIntBetweenPredicate(0, 1000),
+			seriesPageReads:     1,
+			timePageReads:       1,
 		},
 		{
 			name:                "time before results",
 			expectedResultCount: 0,
 			timePredicate:       NewIntBetweenPredicate(-10, -1),
+			seriesPageReads:     1,
```

Comment:
@cyriltovena We could prevent this  single read if we would push down TimeNanos predicates into the SeriesID iterator. 

I don't think we should do this as that PR though 😆